packages/styles/scss/components/tabs-extended/_tabs-extended.scss

```diff
@@ -206,6 +206,16 @@
       }
     }
   }
+  #{$prefix}--accordion__content {
```

Comment:
I'm not understanding the logic here. Both breakpoints have the same spacing. Is the intention for `spacing-05` to apply for `640px` and under? Also, breakpoint values are already mapped, and should be used instead of pixel values.

https://github.com/carbon-design-system/carbon/blob/3cfbc24171c351b475b712edc29a7bfd88f3eb76/packages/grid/scss/_config.scss#L39

Reply:
This was a accidental commit inclusion from another body of work. It's not relevant here and I've taken it out.